### PR TITLE
Shape offsets

### PR DESCRIPTION
**Pull request by greerdv**
## Change summary

Updates shape component and physx collider component to reflect changes around shape offsets and manipulators/component modes.

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Review by SergeyAMZN — APPROVED** [review]

File: content/docs/user-guide/components/reference/shape/capsule-shape.md [hunk 1]
```diff
@@ -25,6 +25,16 @@ The **Capsule Shape** component creates a transparent capsule that's oriented on
...
+| **Translation Offset** | Translation offset of the shape relative to its entity. | Vector3: -Infinity to Infinity | X:`0.0`, Y:`0.0`, Z:`0.0` |
+| **Edit** | Choose the **Edit** button to enter Edit mode. In Edit mode, you can modify the dimensions of the shape in the viewport using the methods outlined in [Edit mode actions](#edit-mode-actions) below. While in Edit mode, the Edit menu in the menu bar displays available actions and hotkeys. To exit Edit mode, choose **Done** in the component interface. |  |  |
+
+## Edit mode actions
+
+The edit mode provides two sub-modes.
+| Mode | Icon | Description
+| - | - | - |
+| **Dimensions** | ![Shape component mode dimensions submode icon](/images/user-guide/components/reference/shape/shape-component-mode-submode-dimensions.svg) | **Left-click** and drag the black handles on the ends of the Capsule Shape to edit the **Height**. By default, each end of the capsule can be separately edited. Hold **Shift** to edit both ends symmetrically. **Left-click** and drag the black handle halfway along the Capsule Shape to edit the **Radius**. |
```
Comment (hultonha) [inline content/docs/user-guide/components/reference/shape/capsule-shape.md:36]: nit: Could add 'situated' to make the phrase slightly clearer potentially?

`"drag the black handle situated halfway along the Capsule Shape to..."`

File: content/docs/user-guide/components/reference/shape/capsule-shape.md [hunk 2]
```diff
@@ -25,6 +25,16 @@ The **Capsule Shape** component creates a transparent capsule that's oriented on
...
+| **Translation Offset** | ![Shape component mode translation offset submode icon](/images/user-guide/components/reference/shape/shape-component-mode-submode-translation-offset.svg) | **Left-click** and drag the linear manipulators or planar manipulators to edit the **Translation Offset**. |
```
Comment (hultonha) [inline content/docs/user-guide/components/reference/shape/capsule-shape.md:37]: nit: Could omit first 'manipulators' so it's just 'linear or planar manipulators'

**Review by hultonha — APPROVED** [review]
Fantastic updates @greerdv! Awesome work 🙂

File: content/docs/user-guide/components/reference/shape/capsule-shape.md (hunk 1 above)
Comment (greerdv) [reply content/docs/user-guide/components/reference/shape/capsule-shape.md]: Yeah, that's nicer, added.

File: content/docs/user-guide/components/reference/shape/capsule-shape.md (hunk 2 above)
Comment (greerdv) [reply content/docs/user-guide/components/reference/shape/capsule-shape.md]: Done

**Pull request merged**